src/js/blight/BlightContainer.jsx

```diff
@@ -5,18 +5,47 @@ export default class BlightContainer extends Component {
     super(props);
     this.totalBlight = this.props.players * 2;
 
-    this.state = { numBlight: this.totalBlight };
+    this.state = { 
+      numBlight: this.totalBlight,
+      blightCardName: "No Effects",
+      blightCardEffect: "",
+      endGameTrigger: 0
+    };
 
     this.handleAddBlight = this.handleAddBlight.bind(this);
     this.handleRemoveBlight = this.handleRemoveBlight.bind(this);
+    this.renderBlightCard = this.renderBlightCard.bind(this);
 
   }
+  renderBlightCard(blights) {
+    if (blights <= 1 && !this.state.endGameTrigger) {
+      var rngBlight = Math.round(Math.random());
```

Comment:
You should make blight card generation scale to more than just 2 cards, since we will be adding more blight cards later on.  Consider using lodash's `shuffle` function. Take a look at `generateFearDeck` in **FearContainer.jsx**. 

Along the same line as scaling, also consider defining a json array that has each blight card as an object. Take a look at **fear-cards.json**, and see **FearContainer.jsx** for how it is used. This way you can just pick one card object, and store it in the state, instead of storing each field of the card.